packages/core/test/globals.js

```diff
@@ -91,11 +94,6 @@ test('globals - ensure circular refs on package compartment global', async (t) =
       const testResult = xyz === globalThis
       console.log(JSON.stringify(testResult, null, 2))
     },
-    context: {
```

Comment:
This is because we invoke every getter, so naturally it triggers the error. I think we can skip this check anyway. `xyz` would not be a `globalThis` reference if the getter could be invoked